capitoli/Quinto_Passo_Primi_Programmi.rst

```diff
@@ -328,7 +330,8 @@ Per ogni esercizio assegnati un punto se riesci a trovare l’errore e un altro
 ::
 
          #Es. 1: 
-	 stampa il nome del tuo cantante preferito.
+         #stampa il nome del tuo cantante preferito.
```

Comment:
```suggestion
         # stampa il nome del tuo cantante preferito.
```

Reply:
Ho effettuato la correzione in tutti i file come da te suggerito.